Updating Enterprise billing (#4569)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/billing.md

As an Enterprise customer, you pay annually via invoice, monthly in arrears for additional usage (if applicable), and may benefit from negotiated usage rates. Please refer to your order form or contract for your specific pricing details, or [contact the account team](https://www.getdbt.com/contact-demo) with any questions. 

> Enterprise plan billing information is not available in the dbt Cloud UI.  Changes are handled through your dbt Labs Solutions Architect or account team manager.

Reviewer: ```suggestion
Enterprise plan billing information is not available in the dbt Cloud UI. Please reach out to your dbt Labs solutions architect or account team manager as they handle all changes.
```
